### PR TITLE
docs: update `pseudo-class` child examples

### LINKED ISSUE #2616: :nth-child - update example to be inclusive
### What information was incorrect, unhelpful, or incomplete?

The example is "NBA championships". This limits the example to men in the USA

### What did you expect to see?

A more inclusive example in terms of gender and geography.

How about tennis champions? That would enable the representation of multiple countries and both men and women. See https://www.espn.com/espn/feature/story/_/id/16371992/tennis-top-20-all (but that is from 2016, so Serena may have moved up to the top)

### URL

(https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)

### PR DESCRIPTION
**Pull request by hirehamir**
fixes #2616.

### CODE REVIEW

**Comment by hamirmahal** [comment]
image

**Comment by hamirmahal** [comment]
image

**Comment by hamirmahal** [comment]
image

**Comment by hamirmahal** [comment]
image

File: live-examples/css-examples/pseudo-class/first-child.html [hunk 1]
```diff
@@ -1,12 +1,21 @@
-<p>NBA players with most championships:</p>
+<p>Accomplished athletes:</p>
```
Comment (estelle) [inline live-examples/css-examples/pseudo-class/first-child.html]:
```suggestion
<p>Track &amp; field champions:</p>
```

File: live-examples/css-examples/pseudo-class/last-child.html [hunk 2]
```diff
@@ -1,12 +1,21 @@
-<p>NBA players with most championships:</p>
+<p>Accomplished athletes:</p>
```
Comment (estelle) [inline live-examples/css-examples/pseudo-class/last-child.html]:
```suggestion
<p>Track &amp; field champions:</p>
```

File: live-examples/css-examples/pseudo-class/nth-child.html [hunk 3]
```diff
@@ -1,12 +1,21 @@
-<p>NBA players with most championships:</p>
+<p>Accomplished athletes:</p>
```
Comment (estelle) [inline live-examples/css-examples/pseudo-class/nth-child.html]:
```suggestion
<p>Track &amp; field champions:</p>
```

**Review by estelle — APPROVED** [review]
Thank you!

**Review by wbamberg — APPROVED** [review]

File: live-examples/css-examples/pseudo-class/first-child.html [hunk 4]
```diff
@@ -1,12 +1,21 @@
-<p>NBA players with most championships:</p>
+<p>Track &amp; field champions:</p>
```
Comment (dipikabh) [inline live-examples/css-examples/pseudo-class/first-child.html:1]:
```suggestion
<p>Track &amp; field champions:</p>
```

**Review by dipikabh — DISMISSED** [review]
making a tiny change to test retriggering the build job

File: live-examples/css-examples/pseudo-class/first-child.html [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
+
```
Comment (dipikabh) [inline live-examples/css-examples/pseudo-class/first-child.html]:
```suggestion
```

**Review by dipikabh — DISMISSED** [review]

**Pull request merged**